finalPrep/annotations.js

```diff
@@ -2,41 +2,62 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+// Declare the function buildABear with the parameters name, age, fur, clothes, specialPower
 function buildABear(name, age, fur, clothes, specialPower) {
+// Within that function, create the variable "greeting" that is assigned to the string below, with the name parameter filling itself in (via interpolation)
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+// Within buildABear, create a variable "demographics" which is assigned to an array containing the bears name and age
   var demographics = [name, age];
+// Within buildABear, create a variable called  powerSaying that is assigned to the string below, with specialPower filling itself in (via string concatenation)
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+// Within buildABear create an object called builtBear that contains the following key value pairs:
   var builtBear = {
+// Key: BasicInfo. Value: the demographics variable from above
     basicInfo: demographics,
+// Key: clothes. Value: the clothes parameter
     clothes: clothes,
+// Key: exterior. Value: the fur parameter
     exterior: fur,
+// Key: cost. Value: the static value 49.99
```

Comment:
Nice work tracking where the values come from- this can be an important and sometimes tedious thing to track in code bases you are looking at for the first time.